docs/src/contrasts.md

```diff
@@ -9,6 +9,13 @@ modeling, `StatsModels` implements a variety of **contrast coding strategies**.
 Each contrast coding strategy maps a categorical vector with $k$ levels onto
 $k-1$ linearly independent model matrix columns.
 
+The following strategies are implemented:
```

Comment:
Thinking about it again, "strategy" sounds a bit unusual in this domain. "Coding scheme" or "coding system" may be more common (e.g. http://www.ats.ucla.edu/stat/r/library/contrast_coding.htm).